kdwsdl2cpp/common/fileprovider.cpp

```diff
@@ -63,6 +65,17 @@ bool FileProvider::get( const QUrl &url, QString &target )
       return true;
   }
 
+  QStringList importPathList = Settings::self()->importPathList();
```

Comment:
const QStringList ...
(so the Q_FOREACH can be ported to c++11 range-for more easily later)